libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -93,6 +94,16 @@ export interface IExtractorProjectConfiguration {
    * The absolute path to the project.
    */
   projectFolder: string;
+  /**
+   * A list of glob patterns to include when extracting this project. If undefined,
+   * all files will be included.
+   */
+  patternsToInclude?: string[];
+  /**
+   * A list of glob patterns to exclude when extracting this project. If undefined,
+   * no files will be excluded.
+   */
+  patternsToExclude?: string[];
```

Comment:
Document which wins if a file covered by both `patternsToInclude` and `patternsToExclude`.